mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/AbstractNioWorker.java

```diff
@@ -94,20 +85,14 @@
     protected final SocketReceiveBufferAllocator recvBufferPool = new SocketReceiveBufferAllocator();
     protected final SocketSendBufferPool sendBufferPool = new SocketSendBufferPool();
     private final DefaultWriteCompletionEventEx writeCompletionEvent = new DefaultWriteCompletionEventEx();
-
-    private final OneToOneRingBuffer ringBuffer;
-    private final Int2ObjectHashMap<Channel> channels;
-    private final AtomicBuffer atomicBuffer = new UnsafeBuffer(new byte[0]);
+    private final Queue<UpstreamMessageEvent> readQueue = new OneToOneConcurrentArrayQueue<>(UDP_CHANNEL_BUFFER_SIZE_PER_WORKER);
```

Comment:
What is UDP_CHANNEL_BUFFER_SIZE_PER_WORKER now intended to mean? Maybe the name should be changed to something like UDP_CHANNEL_READ_QUEUE_SIZE_PER_WORKER